apps/frontend/src/lib/storage/localStorage/user/oauth.ts

```diff
@@ -36,3 +36,15 @@ export const getOAuthAccessToken = async (
 
   return undefined;
 };
+
+export const deleteOAuthAccessToken = async (
```

Comment:
this doesnt actually delete the oauth token from the backup. to do that you probably want to create a new backup type called DELETE_OAUTH_ACCESS_TOKEN or something like that